src/api.ts

```diff
@@ -49,34 +63,41 @@ export let checkLoggedIn = (req, res): boolean => {
     return true;
   } else {
     // Session expired
-    res.status(440).json({ error: "Not logged in!" });
+    res.status(440).json({ error: 'Not logged in!' });
     return false;
   }
-}
+};
 
 // Login
 router.post('/login', async (req, res) => {
-  let members = await db.collection('members').get().then((vals) => {
-    return vals.docs.map(doc => {
-      return doc.data();
+  let members = await db
+    .collection('members')
+    .get()
+    .then((vals) => {
+      return vals.docs.map((doc) => {
```

Comment:
what's docs?

Reply:
I believe `docs` are just the documents from the collection